opl/investigator/check.py

```diff
@@ -42,6 +42,52 @@ def _check_by_min_max(data, value):
     return lower_boundary <= value <= upper_boundary, info
 
 
+def _check_by_max(data, value):
+    logging.debug(f"data={data} and value={value}")
+    mean = statistics.mean(data)
+    lower_boundary = float(mean - (mean - min(data)))
```

Comment:
would `lower_boundary = None` work? or does it freak out
could also put `lower_boundary` to -1 or 0 to indicate there is none

Reply:
I believe None would work. I thought that we are not comparing but we can still report lower boundary.